travis-ci/packer-templates
# Pin chef to an older version when building trusty images

## What is the problem that this PR is trying to fix?
The latest version of chef introduced changes to upgrade behaviour which is breaking our image builds: https://github.com/chef/chef/issues/6549

## What approach did you choose and why?
Pin the version according to the reference [here](https://www.packer.io/docs/provisioners/chef-solo.html#configuration-reference) to the latest one we had successful builds on.

## How can you test this?
Run an image builds and see that the upgrade succeeds.
Test run here: https://travis-ci.org/travis-infrastructure/packer-build/jobs/296215864#L827
(the build currently fails due to an unrelated problem)
